### PR TITLE
Selenium e2e tests

### PR DESCRIPTION
**Pull request by alsuren**
Test with `watchexec --watch=test/internet-explorer/ -- ./test/scripts/internet-explorer-bs.sh prod` (or just `./test/scripts/internet-explorer-bs.sh prod` if you don't need a watcher.)

with `./test/scripts/local-bs-connect.sh` running in a background tab.

Includes test for [AB#1766](https://dev.azure.com/futurenhs/75407963-f812-43e5-a734-1059bbc036a3/_workitems/edit/1766) - Workspace homepage 

## Testing information

- Is there any special setup required?

- Test plan?

## Test:

- [x] Tested locally

- Platforms/Browsers:

  - [ ] Google Chrome (latest version)

  - [ ] Internet Explorer 11

  - [ ] Edge

- Accessibility:

  - [ ] Screen Reader

  - [ ] Keyboard Navigation

  - [ ] Magnification

  - [ ] Contrast

  - [ ] Text size 200%

  - [ ] Touch target areas (Mobile)

  - [ ] Landscape (Mobile)

### CODE REVIEW

File: test/internet-explorer/login.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  };
+  let driver = new webdriver.Builder()
+    .usingServer(browserstackURL)
+    .withCapabilities(capabilities)
+    .build();
+
+  it("should render login page", async () => {
+    await driver.get(`${baseUrl}/admin/create-workspace`);
+    const emailInput = await driver.findElement(webdriver.By.css("#email"));
+    const passwordInput = await driver.findElement(webdriver.By.css("#email"));
```
Comment (frigus02) [inline test/internet-explorer/login.js:47]: That looks like a typo:
```suggestion
    const passwordInput = await driver.findElement(webdriver.By.css("#password"));
```

**Review by frigus02 — COMMENTED** [review]
Nice. This looks fine to me 👍

File: test/.env.example [hunk 2]
```diff
@@ -1,3 +1,8 @@
 
 BROWSERSTACK_USERNAME=
 BROWSERSTACK_ACCESS_KEY=
+
+TEST_LOGIN_EMAIL_ADDRESS=
+TEST_LOGIN_PASSWORD=
+# The "Selenium Testing" workspace ID.
```
Comment (alsuren) [inline test/.env.example:7]:
```suggestion
# The "Selenium Testing" workspace ID on production.
```

**Review by matt-paul — APPROVED** [review]

File: test/internet-explorer/login.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+const assert = require("assert");
+
+const fs = require("fs");
+
+/**
+ * @param {string} name
+ * @returns {string}
+ */
+function env(name) {
+  let value = process.env[name];
```
Comment (frigus02) [inline test/internet-explorer/login.js:14]: I think we could use `const` here.

**Review by frigus02 — APPROVED** [review]

**Pull request merged**